util/misc.go

```diff
@@ -132,21 +133,19 @@ func SyntaxWarn(err error) error {
 	return parser.ErrParse.GenWithStackByArgs(syntaxErrorPrefix, err.Error())
 }
 
-const (
+var (
 	// InformationSchemaName is the `INFORMATION_SCHEMA` database name.
-	InformationSchemaName = "INFORMATION_SCHEMA"
-	// InformationSchemaLowerName is the `INFORMATION_SCHEMA` database lower name.
-	InformationSchemaLowerName = "information_schema"
+	InformationSchemaName = model.CIStr{O: "INFORMATION_SCHEMA", L: "information_schema"}
 	// PerformanceSchemaName is the `PERFORMANCE_SCHEMA` database name.
-	PerformanceSchemaName = "PERFORMANCE_SCHEMA"
-	// PerformanceSchemaLowerName is the `PERFORMANCE_SCHEMA` database lower name.
-	PerformanceSchemaLowerName = "performance_schema"
+	PerformanceSchemaName = model.CIStr{O: "PERFORMANCE_SCHEMA", L: "performance_schema"}
+	// MetricSchemaName is the `METRIC_SCHEMA` database name.
+	MetricSchemaName = model.CIStr{O: "METRIC_SCHEMA", L: "metric_schema"}
```

Comment:
Prefer using `model.NewCIStr("...")` to initialize the variable.

Reply:
I prefer too, but can't do this.....